### ISSUE TITLE
We need a researcher creature for keepers

### ISSUE THREAD

**Issue by Danimal696** [OpenDungeons/OpenDungeons#1083]

**Comment by hwoarangmy** [comment]
Don't you like the lizardman ? Have you got something in mind ?

**Comment by Danimal696** [comment]
Yes ;)

**Comment by hwoarangmy** [comment]
Looks like a teasing issue ^^

**Comment by Danimal696** [comment]
totally, besides a wand, what would be a good weapon for an evil mage?

**Comment by akien-mga** [comment]
A crooky staff, but that would likely be in replacement of the wand, so if you already have the latter it's not a good fit :)
A kris maybe? https://en.wikipedia.org/wiki/Kris

**Comment by Danimal696** [comment]
DEAL!! creepy sacrificial knifes for him from here on.

**Comment by Danimal696** [comment]
Meet the "Custist" AKA Dark Priest from ancient beast, who sports an unhealthy passion for sacrificies, praying in hidden dark temples and researching forgotten rituals; keep away all your virgins from him, you have been warned.

And so this bad boy is mostly ready, only rigging and animating left:
![wip](https://cloud.githubusercontent.com/assets/8299756/12558029/e22c2368-c38d-11e5-92fe-de4f74941feb.png)

![wip2](https://cloud.githubusercontent.com/assets/8299756/12558037/ecb28a70-c38d-11e5-8cc5-b2ff279a8f75.png)
![wip3](https://cloud.githubusercontent.com/assets/8299756/12558038/ee64822e-c38d-11e5-9c59-ac3f638cffa5.png)

**Comment by Danimal696** [comment]
I think this creature to be an exception to the tiers, he will be a tier1 who can grow to be a match to any tier, at first he will be pretty weak, only a range 1 physical attack at lv1, but once he start learning magic skills he will became your artillery of choosing. His stats could go like this:

Class: Cultist => Cristal Cannon
Hp: low 
Def: low to phys, med to ele, high to magic
Speed: low
Skills:
Lv1: Stab
Lv5: Magic bolt - low Magic dmg
Lv8: Slow curse
Lv10: Healing
Lv15: Fireball - medium Elemental dmg, small area effect, short stun effect
Lv18: Str down Curse
Lv20: Poison thorns - Phys dmg+ poison effect 
Lv25: Magic Explosion - high Magic dmg - Big area - long cooldown
Lv30: Freeze - enemy totally disabled for a short time

Thats just a draft, but the idea is he is a growing ranged powerhouse with some support skills, but needs to be away from harms way at all cost. Phys ranged like archers and fast melee would be his worst enemies.

**Comment by Bertram25** [comment]
I love the monster and the concept. You have something interesting here.
What I like is that you have some kind of monster you try to protect from harm, which adds some spice to the game. On the other hand, this will turn it into some kind of "mascot", which is a good and bad thing as it is already the mascot of Ancient Beast.
But well, why not? I'd say let's try.

**Comment by Danimal696** [comment]
Sadly AB is frozen, but its monsters are amazing, i will be using more of them in the future. The wivern and many others are really good and have all the character ours currently lacks.

**Comment by Bertram25** [comment]
Well, if Ancient Beast is frozen, I hope they don't mind if we reuse their creatures (with due credits & licensing)

**Comment by akien-mga** [comment]
I'd say we shouldn't refrain from using Ancient Beast art for now. If they do happen to resurrect and it appears annoying to have the same art in both games, we'll give them the priority and try to find replacements ;)

**Comment by Danimal696** [comment]
all of them are cc-by-sa and Dread knigth is friendly enought to have already allowed it :), hes even happy of someone finishing them since he wants to turn 3d some day. And we can throw jokes like the cultist claiming he has his own game and he is some kind of pokemons trainer

**Comment by Bertram25** [comment]
Even cooler then. :)

**Comment by hwoarangmy** [comment]
Indeed, it fits well for a research creature ^^

**Comment by Bertram25** [comment]
@Danimal696 Do you think this is still relevant for 0.7.0? Or do you need more time?

**Comment by Danimal696** [comment]
not really relevant as long as other creature researches for the time being, yes, rigging is coming pretty slowly...

**Comment by Bertram25** [comment]
Ok, setting it to Later, as it'll come when it'll come.

**Comment by Danimal696** [comment]
Just pending of #1167 resolving

https://www.youtube.com/watch?v=xyiEAO7U3x8

**Comment by Danimal696** [comment]
I have planned to use this rig as a base to build upon, so i also added a few new torture animations that will fit all new humanoid characters and use the new models from eugeneloza, they are:
- Torture in electric chair
- Torture by putting in a small cage
- Torture over a table

You can see them as the last three animations; Im drooping the motion capture for now since its pretty limited in variety, these should speed up new characters a bit but animation quality will drop somehow as a tradeoff.

**Comment by Loobinex** [comment]
I like the look of the model. What looks a bit silly to me though is the stretching of the middle part of the robe when he walks/runs. (the piece with the yellow crosses). This makes it look like he's wearing spandex.

Would it be possible to make that bit of the texture not stretch when he pulls his legs apart, but instead stretch the brown bit of the robe?

**Comment by Danimal696** [comment]
Not without major changes to model, armature and animations, i would spend roughly the same time correcting this than animating a new model. But i appreciate the feedback, i wont be repeating the same mistake again, after all im still learning animating and blender stuff.

**Comment by hwoarangmy** [comment]
Indeed, the stretching of the yellow crosses looks a bit weird. But I guess it won't be that visible ingame so it might not be worth fixing.

**Comment by Danimal696** [comment]
the animation for mele attack is the most visible, but since we still dont ramdomize attack animations ill just declare the spell attack one as the default
